Makefile.kokkos

```diff
@@ -265,11 +265,16 @@ else
     KOKKOS_INTERNAL_OPENMP_FLAG := -fopenmp
   endif
 endif
-ifeq ($(KOKKOS_INTERNAL_COMPILER_INTEL_CLANG), 1)
-  KOKKOS_INTERNAL_OPENMPTARGET_FLAG := -fiopenmp -Wno-openmp-mapping
-else ifeq ($(KOKKOS_INTERNAL_COMPILER_CLANG), 0)
-  #Assume GCC
-  KOKKOS_INTERNAL_OPENMPTARGET_FLAG := -fopenmp -foffload=nvptx-none
+
+ifeq ($(KOKKOS_INTERNAL_USE_OPENMPTARGET), 1)
```

Comment:
This is not really necessary, right? You are only setting `KOKKOS_INTERNAL_OPENMPTARGET_FLAG` anyway.

Reply:
Yeah we don't really need it, I just wanted to avoid multiple checks in that loop. I can edit it if thats preferred.